components/kyma-environment-broker/internal/process/deprovisioning/initialisation.go

```diff
@@ -105,7 +110,7 @@ func setAvsIds(deprovisioningOperation *internal.DeprovisioningOperation, provis
 	}
 }
 
-func (s *InitialisationStep) checkRuntimeStatus(operation internal.DeprovisioningOperation, instance *internal.Instance, log logrus.FieldLogger) (internal.DeprovisioningOperation, time.Duration, error) {
+func (s *InitialisationStep) checkRuntimeStatus(operation internal.DeprovisioningOperation, instance *internal.Instance, planID string, log logrus.FieldLogger) (internal.DeprovisioningOperation, time.Duration, error) {
```

Comment:
the instance contains plan ID, you don't need to introduce a new parameter

Reply:
Thanks!